### PR TITLE
feat(poetry): use poetry v1.0.0 in Dockerfile

### LINKED ISSUE #4843: Poetry.lock updates failing
**What Renovate type are you using?**

Renovate self-hosted for self-hosted Gitlab

**Describe the bug**

Yesterday renovate stopped updating our `poetry.lock` file located in a subfolder `api/`. This has been working fine for months and we didn't change any configuration, just applied automatic updates to renovate.

**Did you see anything helpful in debug logs?**

```
 INFO: Failed to update api/poetry.lock file (repository=alpha/alpha, dependencies=pytest-mock, branch=ren/pytest-mock-1.x)
       "err": {
         "killed": false,
         "code": 1,
         "signal": null,
         "cmd": "poetry update --lock --no-interaction pytest-mock",
         "stdout": "Creating virtualenv alpha-api-py3.8 in /home/ubuntu/.cache/pypoetry/virtualenvs\nThe virtual environment was not created successfully because ensurepip is not\navailable.  On Debian/Ubuntu systems, you need to install the python3-venv\npackage using the following command.\n\n    apt-get install python3-venv\n\nYou may need to use sudo with that command.  After installing the python3-venv\npackage, recreate your virtual environment.\n\nFailing command: ['/home/ubuntu/.cache/pypoetry/virtualenvs/alpha-api-py3.8/bin/python', '-Im', 'ensurepip', '--upgrade', '--default-pip']\n\n",
         "stderr": "",
         "message": "Command failed: poetry update --lock --no-interaction pytest-mock\n",
         "stack": "Error: Command failed: poetry update --lock --no-interaction pytest-mock\n\n    at ChildProcess.exithandler (child_process.js:294:12)\n    at ChildProcess.emit (events.js:198:13)\n    at ChildProcess.EventEmitter.emit (domain.js:448:20)\n    at maybeClose (internal/child_process.js:982:16)\n    at Socket.stream.socket.on (internal/child_process.js:389:11)\n    at Socket.emit (events.js:198:13)\n    at Socket.EventEmitter.emit (domain.js:448:20)\n    at Pipe._handle.close (net.js:606:12)"
       }
 INFO: files committed (repository=alpha/alpha, dependencies=pytest-mock, branch=ren/pytest-mock-1.x)
 INFO: PR created (repository=alpha/alpha, dependencies=pytest-mock, branch=ren/pytest-mock-1.x)
       "pr": 3650
```

### PR DESCRIPTION
**Pull request by trim21**
poetry has released 1.0.0 with some breaking change.

config key `settings.virtualenvs.in-project` has been renamed to `virtualenvs.in-project`

Closes #4843

### CODE REVIEW

**Comment by rarkins** [comment]
@Trim21 thanks for your PR. are there any reference public repos we can use to test this against via CLI?

**Comment by Trim21** [comment]
@rarkins https://github.com/Trim21/pol/tree/3feba248fe0fb7498fbba25606db88427118361c

i'm using poetry 1.0.0 and get no lockfile update

pr by WhiteSrouce app https://github.com/Trim21/pol/pull/105

File: Dockerfile [hunk 1]
```diff
@@ -192,11 +192,10 @@ RUN pip install --user pipenv
 
 # Poetry
 
-RUN curl -sSL https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py | python
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
```
Comment (ViceIce) [inline Dockerfile]: Shouldn't we install a specific version like other tools?

```suggestion
RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python --version 1.0.0
```

Of cause `1.0.0` should com from env

**Review by viceice — DISMISSED** [review]
Repo was moved

https://github.com/python-poetry/poetry/blob/master/CHANGELOG.md#100---2019-12-12

File: Dockerfile (hunk 1 above)
Comment (Trim21) [reply Dockerfile]: Won't work, it will print python version

Comment (rarkins) [reply Dockerfile]: Can we do like in the dedicated docker image and get it then run it?

Comment (ViceIce) [reply Dockerfile]: See updated suggestion, should work added the `-` to force `stdin` mode

**Review by viceice — APPROVED** [review]

**Comment by ViceIce** [comment]
Strange, having problems to build the full image. it hangs on downloading the erlang packages

Anybody able to successfully build the image?

**Comment by ViceIce** [comment]
master already fails
image

**Review by viceice — APPROVED** [review]

**Pull request merged**